frontend/app/assets/stylesheets/spree/frontend/variables/variables.scss

```diff
@@ -1,11 +1,11 @@
 $primary-background: #ffffff;
 $font-color: black;
-$footer-background: $primary-background;
-$footer-font-color: $font-color;
-$meganav-background: $primary-background;
-$header-background: $primary-background;
 $header-font-color: $font-color;
-$meganav-font-color: $header-font-color;
+$header-background: $primary-background;
+$meganav-font-color: $font-color;
```

Comment:
I think this should default to `$header-font-color`

Reply:
No, it shouldn't